hazelcast-spring/src/main/java/com/hazelcast/spring/HazelcastNamespaceHandler.java

```diff
@@ -40,6 +40,10 @@ public void init() {
                 "set",
                 "list",
                 "executorService",
+                "durableExecutorService",
```

Comment:
`durableExecutorService` and `scheduledExecutorService` are not needed here or they should be added to `hazelcast-spring-3.10.xsd`. This list is for namespaces which allow the user to retrieve instances of distributed objects using just XML config, without calling methods on `HazelcastInstance`. 
For the current list, see:
https://github.com/hazelcast/hazelcast/blob/master/hazelcast-spring/src/main/resources/hazelcast-spring-3.10.xsd#L1608-L1836

This is different than a configuration for a distributed object which is created by the user later:
https://github.com/hazelcast/hazelcast/blob/master/hazelcast-spring/src/main/resources/hazelcast-spring-3.10.xsd#L41-L1440

So either add them to the XML section in the first link or remove them from here.

Reply:
Added in XML and removed from here.

Reply:
It needs to be in both XML and here or neither in XML nor here. Having it in just one of these two places doesn't work.

Reply:
Argh, added in both.